numpy/core/tests/test_numeric.py

```diff
@@ -2748,3 +2748,7 @@ def test_zero_dimension(self):
         td = np.tensordot(a, b, (1, 0))
         assert_array_equal(td, np.dot(a, b))
         assert_array_equal(td, np.einsum('ij,jk', a, b))
+        # Test resolution to issue #12130
```

Comment:
Could be spelled simply "gh-12130"

Reply:
Better to do it this way - the tests are littered with references to different historical issue trackers - it's always nice to know which issue tracker they were referring to, just in case we change again.